content/400-reference/200-api-reference/300-environment-variables-reference.mdx

```diff
@@ -0,0 +1,187 @@
+---
+title: 'Environment variables reference'
+metaTitle: 'Prisma environment variables reference'
+metaDescription: 'This page gives an overview of all environment variables avalaible for use.'
+tocDepth: 2
+---
+
+## Overview
+
+This document describes different environment variables and their use cases.
+
+### DATABASE_URL
+
+`DATABASE_URL` is used to connect to your database. The url format can differ depending on the database it is connecting to.
+Prisma currently supports 4 types of database connectors, of which 3 use the `DATABASE_URL` environment variable. (SQLite sources a file path)
+
+- PostgreSQL
+- MySQL
+- SQLite
+- SQLServer
+
+Example using a PostgreSQL connector:
+
+```js file=.env
+DATABASE_URL = 'postgresql://USER:PASSWORD@HOST:PORT/DATABASE'
+```
+
+See [Connection URLs](../database-reference/connection-urls) for a breakdown of each database connector.
+
+### DEBUG
+
+`DEBUG` is used to enable debugging output in the `Prisma Client`.
+
+Example setting `Prisma Client` level debugging output:
+
+```terminal
+# enable only `prisma-client`-level debugging output
+export DEBUG="prisma-client"
```

Comment:
Seems that we have a mix of `prisma-client` and `prisma:client` so maybe not the best example until we fix this?

Reply:
> Seems that we have a mix of prisma-client and prisma:client 

What do you mean with that Joel? `prisma-client` is the only way that's documented at the moment.

Reply:
I mean in our code with have a mix of these 2 values 🙈

Reply:
Ok, then it sounds like something internal that should be cleaned up but on a "developer surface" level, we never talked about `prisma:client`.

Reply:
Therefore IMO it's fine to use this example